indra/newview/llviewertexture.cpp

```diff
@@ -559,7 +559,10 @@ void LLViewerTexture::updateClass()
         // lower discard bias over time when free memory is available
         if (sDesiredDiscardBias > 1.f && over_pct < 0.f)
         {
-            sDesiredDiscardBias -= gFrameIntervalSeconds * 0.01f;
+            static LLCachedControl<F32> high_mem_discard_increment(gSavedSettings, "RenderHighMemMinDiscardDecrement", .1f);
```

Comment:
Typo:
high_mem_discard_**increment**

Also won't it be better to make this dynamic? The more memory is free the faster bias falls?

Reply:
See line below, it is faster when more memory is free (over_pct is negative), fixing typo...

Reply:
Sorry, somehow missed that.